g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SourceServiceImpl.java

```diff
@@ -34,6 +33,8 @@
 public class SourceServiceImpl implements SourceService {
 	
 	private final static Logger LOGGER = LoggerFactory.getLogger(SourceService.class);
+	private final static String MD = "MD";
+	private final static String HTML = "HTML";
```

Comment:
define a string array for the content types.

Reply:
done